pkg/controllers/raycluster_webhook.go

```diff
@@ -66,17 +68,51 @@ var _ webhook.CustomValidator = &rayClusterWebhook{}
 func (w *rayClusterWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	rayCluster := obj.(*rayv1.RayCluster)
 
-	if !pointer.BoolDeref(w.Config.RayDashboardOAuthEnabled, true) {
-		return nil
-	}
-
 	rayclusterlog.V(2).Info("Adding OAuth sidecar container")
 
-	rayCluster.Spec.HeadGroupSpec.Template.Spec.Containers = upsert(rayCluster.Spec.HeadGroupSpec.Template.Spec.Containers, oauthProxyContainer(rayCluster), withContainerName(oauthProxyContainerName))
+	if pointer.BoolDeref(w.Config.RayDashboardOAuthEnabled, true) {
```

Comment:
In @KPostOffice's PR (https://github.com/project-codeflare/codeflare-operator/pull/498) he updated from `pointer` to `ptr`. Noticing here we're using `pointer` and wanted to ask if we need to update it here too?

Reply:
I believe this ptr update will be handled in the [cert-controller pr](https://github.com/project-codeflare/codeflare-operator/pull/533/files) :)